classes/rule.php

```diff
@@ -350,6 +374,59 @@ public function process(&$request, $mod, $currenttime) {
         return true;
     }
 
+    /**
+     * Checks rules for each request and writes the subscriptions for one user.
```

Comment:
Let's update this comment slightly to reflect the fact that we're only considering one specific request and mod for the given user.